### PR TITLE
Fix bugs in run_danesfield.

### PR DESCRIPTION
**Pull request by danlipsa**
- add the <basename>.png as texture file name
- use node which is the name of the updated node interpreter. nodejs
  is the older program installed with npm.

### CODE REVIEW

**Comment by danlipsa** [comment]
@jacobderosa @dmjoy Please review.

**Comment by dmjoy** [comment]
LGTM apart from the one suggestion.

File: tools/tiler.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def get_obj_texture_file_name(filename):
     Given an OBJ file name return a file name for the texture by removing .obj and
     adding .png
     """
-    file_no_ext = os.path.splitext(filename)[0]
+    file_no_ext = os.path.basename(filename)
     return file_no_ext + ".png"
```
[the remark this answers is not in the document]
Comment (jacobderosa) [reply tools/tiler.py:26]: That change wasn't in PR #17, what's the conflict?

**Comment by dmjoy** [comment]
@jacobderosa here's the conflict (doesn't seem related to the line above as I initially thought):
`
<<<<<<< fix_run_danesfield
        cmd_args = ["node", "/3d-tiles-tools/tools/bin/3d-tiles-tools.js",
=======
        cmd_args = ["nodejs", "/3d-tiles-validator/tools/bin/3d-tiles-tools.js",
>>>>>>> master
`

**Comment by jacobderosa** [comment]
I applied these changes in PR #19 @danlipsa @dmjoy

**Pull request closed without merging**